app/controllers/meetings/download/copy.js

```diff
@@ -7,7 +7,7 @@ import { copyStringToClipboard } from '../../../utils/copy-string-to-clipboard';
 export default class MeetingsDownloadCopyController extends Controller {
   @action
   async copyToClipboard(text) {
-    await copyStringToClipboard({ html: text });
+    await copyStringToClipboard({ html: text }, true);
```

Comment:
this function only takes a single object argument, unless some git shenanigans happened and I'm not seeing the changes you made to it

you maybe meant to add `renderHtmlToPlain: true`? but that's also true by default, so I'm not sure this is really doing anything

Reply:
Oof, I must have been tired when I put this in...